### ISSUE TITLE
Invalid comment?

### ISSUE THREAD

**Issue by ibtaylor** [strasdat/Sophus#119]
https://github.com/strasdat/Sophus/blob/3f84e42a7a796bb6c9a1f57f5708911459a49221/sophus/se3.hpp#L128-L130
https://github.com/strasdat/Sophus/blob/3f84e42a7a796bb6c9a1f57f5708911459a49221/sophus/so3.hpp#L164-L166

Was the comment just copy-pasted from `so3.hpp` to `se3.hpp`? Trying to understand how either of them are SU(2) representation.

**Comment by NikolausDemmel** [comment]
Could be a typo, but it also makes some sense, since so(3) is internally represented as SU(2) and se(3) is represented as  SU(2) x R^3, and the SU(2)  part is what differentiates it from other choices.

But I agree that this could be made clearer.

**Comment by ibtaylor** [comment]
```cpp
SO3<S>::internalJacobian() -> Matrix<S,4,3>
SE3<S>::internalJacobian() -> Matrix<S,7,6>
```
Can both of these can still be considered SU(2) representation?

I find plenty of articles on the net talking about the connection between _SO(3)_ and _SU(2)_, but I find nothing about _SE(3)_ and _SU(*)_. I'm just trying to understand the code/math. Sort of related to #118

**Comment by NikolausDemmel** [comment]
SE(3) objects are stored as translation and rotation, where translation an R^3 element and orientation is stored as a an SU(2) element. Therefore, SE(3) is represented as SU(2) x R^3, but group operation (pose concatination) is defined such that it corresponds to the SE(3) pose product. In the same way, you can represent SE(3) objects as SO(3) x R^3 with a corresponding group operation.

**Comment by strasdat** [comment]
I believe this is addressed by PR #133, which replaces ``internalJacobian`` by ``Dx_exp_x_at_0``.
